doc/src/main/tut/usage.md

```diff
@@ -205,9 +205,17 @@ A main method that uses `decline` for argument parsing would look something like
 
 ```scala mdoc:to-string
 def main(args: Array[String]) = tailCommand.parse(args, sys.env) match {
-  case Left(help) =>
+
+  case Left(help) if (!help.errors.isEmpty)=>
```

Comment:
Minor, but: let's drop the parens, and switch the order of these two cases so we don't need the `!` either?